demo/demo-spring-boot-discovery/demo-spring-boot-zuul-proxy/pom.xml

```diff
@@ -30,6 +30,17 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-zuul</artifactId>
+			<version>1.3.0.RELEASE</version>
```

Comment:
can we use a single spring cloud parent instead of managing all the versions ourselves?